packages/p2p/src/components/my-profile/block-user/block-user.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { observer } from 'mobx-react-lite';
+import { useStores } from 'Stores';
+import { DesktopWrapper, MobileFullPageModal, MobileWrapper } from '@deriv/components';
+import BlockUserModal from 'Components/block-user/block-user-modal';
+import BlockUserTable from 'Components/my-profile/block-user/block-user-table/block-user-table';
+import SearchBox from 'Components/search-box';
+import { my_profile_tabs } from 'Constants/my-profile-tabs';
+import debounce from 'lodash.debounce';
+import { localize } from 'Components/i18next';
+
+const BlockUser = () => {
+    const { general_store, my_profile_store } = useStores();
+
+    const loadBlockedAdvertisers = debounce(search => {
+        my_profile_store.setSearchTerm(search.trim());
+        my_profile_store.loadMoreBlockedAdvertisers();
+    }, 200);
```

Comment:
Set debounce duration to `200`ms instead (usually its set to `1000`ms), it *should* be lenient to reload the blocked advertisers list much more often since we are not using websockets to re-request for the list of blocked advertisers compared to the buy-sell search box `loadMoreItems` action